src/test/com/homework/week9/invoice/service/InvoiceServiceTest.java

```diff
@@ -0,0 +1,47 @@
+package com.homework.week9.invoice.service;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class InvoiceServiceTest {
+    private InvoiceService testObject;
+
+    @BeforeEach
+    public void makeInvoceObject() {
+        testObject = new InvoiceService();
```

Comment:
please create MockedInvoiceReader class and provide fixed list of invoices to not read the real invoices from txt file

Reply:
and for each test provide list of new invoices and make assertions

Reply:
nor sure if understand what should do MockedInvoiceReader .